src/add-archival-events/src/api.ts

```diff
@@ -27,6 +43,15 @@ export const isRecordInAuditLog = async (
     return { exists: false, err: true }
   }
 
+  // Create audit log message if one does not already exist
+  if (messageResult === undefined) {
```

Comment:
We can simplify this to `!messageResult`

Reply:
done in e88a4ec11b0b94905b3e41c2ebe59cc0fa02475b